.golangci.yml

```diff
@@ -15,7 +15,6 @@ linters:
   disable-all: true
   enable:
     - durationcheck
-    - exportloopref
```

Comment:
Removed as per `ERRO [linters_context] exportloopref: This linter is fully inactivated: it will not produce any reports. `